obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -267,6 +267,65 @@ static void DeleteOldestFile(const char* location, unsigned maxLogs)
 #include <unistd.h>
 #endif
 
+outdated_driver_error * outdated_driver_error::inst = nullptr;
+
+outdated_driver_error * outdated_driver_error::instance()
+{
+	if(inst == nullptr)
+	{
+		inst = new outdated_driver_error();
+	}
+	return inst;
+}
+
+void outdated_driver_error::set_active( bool state) 
+{
+	if (state) {
+		if (!lookup_enabled)
+		{
+			line_1 = "";
+			line_2 = "";
+		}
+		lookup_enabled++;
+	} else {
+		lookup_enabled--;
+	}
+}
+
+std::string outdated_driver_error::get_error(const char* base_error)
+{
+	if (std::string(base_error).find("Function not implemented") == std::string::npos)
```

Comment:
I would add a check here since base_error can be null

Reply:
check added.